src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_skadi.cpp

```diff
@@ -96,6 +96,7 @@ static Position SkadiPosition[] =
 };
 
 #define EMOTE_IN_RANGE   "Skadi the Ruthless is within range of the harpoon launchers"
+#define EMOTE_DRAKE_BREATH "Grauf takes a deep breath"
```

Comment:
NO! This should be handled in creature_text always.

Reply:
I'll add the emote and use existing ones in creature_text.

Reply:
Now uses creature_text appropriately